schemas/SHARE/v2/internal/schema.yaml

```diff
@@ -0,0 +1,324 @@
+!python/share.schema.Schema
+name: SHARE v2 Interal
+types:
+- !python/share.schema.SchemaType
+  - name: AgentWorkRelation
+  - description: null
+  - fields: !!omap
+    - type: !python/share.schema.SchemaField
+        choices:
+        - unknown
+        - contributor
+        - creator
+        - principal investigator
+        - principal investigator contact
+        - funder
+        - host
+        - publisher
+        db_index: true
+        max_length: 255
+        type: django.db.models.CharField
+    - cited_as: !python/share.schema.SchemaField
+        blank: true
+        type: django.db.models.TextField
+    - agent: !python/share.schema.SchemaField
+        related_name: work_relations
+        to: Agent
+        type: django.db.models.ForeignKey
+    - creative_work: !python/share.schema.SchemaField
+        related_name: agent_relations
+        to: CreativeWork
+        type: django.db.models.ForeignKey
+    - order_cited: !python/share.schema.SchemaField
+        'null': true
+        type: django.db.models.PositiveIntegerField
+- !python/share.schema.SchemaType
+  - name: CreativeWork
+  - description: null
+  - meta:
+      verbose_name_plural: Creative Works
+  - fields: !!omap
+    - type: !python/share.schema.SchemaField
+        choices:
+        - creative work
```

Comment:
Might be out of scope for this PR, but should we use `unknown` for the "base" work type? Like in AgentWorkRelation.type above.